arcgis-ios-sdk-samples/AppDelegate.swift

```diff
@@ -143,6 +146,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UISplitViewControllerDele
         return nil
     }
     
+    // MARK: - User data
+    
+    /// Creates a dictionary to store the user's favorited samples.
+    func createUserDefaults() {
+        if UserDefaults.standard.dictionary(forKey: "favorites") == nil {
```

Comment:
Can the string "favorites" be made a static property, so you don't have to use the same string twice?